Unix/samples/Providers/Test_Indication/impl.c

```diff
@@ -446,8 +446,8 @@ _Use_decl_annotations_
         case PostBehavior_Subcontext:
             startfireindication = MI_FALSE;
             break;
-        case PostBehavior_Enablecontext_PostOnCalllthread:
-        case PostBehavior_Subcontext_PostOnCalllthread:
+        case PostBehavior_Enablecontext_PostOnCallThread:
```

Comment:
The split of `Callthread` to `CallThread` is thrown in, since we're already changing the token